crypto/hash/hash_test.go

```diff
@@ -266,7 +257,7 @@ func TestKeccak(t *testing.T) {
 func BenchmarkComputeHash(b *testing.B) {
 
 	m := make([]byte, 32)
-	rand.Read(m)
+	_, _ = rand.Read(m)
```

Comment:
In these bench functions, if randomness fail, we would measure the hash of all zeros instead of a random input. 
I'll add the check to avoid confusion.